IdentityCore/src/broker_operation/request/interactive_token_request/MSIDBrokerOperationInteractiveTokenRequest.m

```diff
@@ -34,10 +34,7 @@ @implementation MSIDBrokerOperationInteractiveTokenRequest
 
 + (void)load
 {
-    if (@available(macOS 10.15, *))
```

Comment:
Minimum version for macOS is 10.15 (101500) so all checks like this have been removed as they are now always true